src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -1451,11 +1706,17 @@ void Hole::onChanged(const App::Property* prop)
         }
     }
     else if (prop == &Tapered) {
-        if (Tapered.getValue())
+        if (Threaded.getValue()) {
+            TaperedAngle.setValue(getThreadProfileAngle());
+            TaperedAngle.setReadOnly(true);

```

Comment:
While it's nice to have the taper angle from the standards now available, it is not possible anymore to use custom angles in combination with threads. Here we need to find a solution as well.

Reply:
it could be left as editable, but thread standards have clearly defined taper angles, having a conical thread with a custom angle is not a standard thread.

Reply:
Yes, we did not have standard conical threads. We only had custom conical threads. And those had issues too, before you fixed it. Thank you for that!

But here we have the same situation as with the ISO 2306 core holes: You just removed this existing feature in favour of your new ones. I think this is not OK.

A solution may be an additional property (defaulting to true for older documents) "CustomTaplerAngle", similar to the custom clearance we already have. My preference is put this part in a dedicated PR, though.